pallets/vault-registry/src/lib.rs

```diff
@@ -29,7 +29,7 @@ use sp_std::{
 };
 
 pub use currency::Amount;
-pub use default_weights::WeightInfo;
+pub use default_weights::{WeightInfo, SubstrateWeight};
```

Comment:
Is this import necessary? I don't see it being used.

Reply:
This will be used  for the runtime in pendulum:

https://github.com/pendulum-chain/pendulum/blob/pendulum-release/runtime/foucoco/src/lib.rs#L445
https://github.com/pendulum-chain/pendulum/blob/pendulum-release/runtime/foucoco/src/lib.rs#L789
https://github.com/pendulum-chain/pendulum/blob/pendulum-release/runtime/foucoco/src/lib.rs#L804

Reply:
i actually forgot the other pallets, so I'm adding them now